### ISSUE TITLE
Remote Node

### ISSUE THREAD

**Issue by mukw-labs** [kvj/hass_Mtastic_MQTT#6]
Possibly my ignorance here.
But if I am configuring the MQTT topic to a specific device, doesn't that mean that device would need an MQTT connection.
Therefore if I have a mobile device that has no mqtt connection, I won't see any of the info as it will be posted as a protobuf on another devices topic?

**Comment by kvj** [comment]
Your mobile device could provide an MQTT connection to the Meshtastic node. There's a checkbox in configuration for that

**Comment by mukw-labs** [comment]
But doesn't that negate the use of a Lora based device. If I have a mobile device then I have no need for the Meshtastic.
I am not 100% sure on the MQTT of Meshtastic, but I'm assuming that a remote node information is in the protobuf of another node as they are relaying the information.

**Comment by jhbruhn** [comment]
I think what mukowman meant was the possibility to also process messages from other nodes which are published on the same channel the integration is listening on. This makes a lot of sense as it will allow users to integrate the whole meshtastic mesh into HA, not only the local MQTT bridge device. This could be combined with #9

**Comment by hassos246** [comment]
This actually works already just change the Node ID in the Node ID field. The topic should be the same. Not sure about the stat topic.

**Comment by mukw-labs** [comment]
Wouldn't that require you to know which node is sending the traffic?

**Comment by hassos246** [comment]
Yes it does. But you can see the user id for any node connected to the mesh. I monitor my two non mqtt nodes this way thru my mqtt node via this integration.
